.dockerignore

```diff
@@ -23,6 +23,7 @@ frontend/lib/queries/__generated__/
 querybuilder.js
 querybuilder.js.map
 react-loadable.json
+.DS_Store
```

Comment:
@romeboards we want our `.dockerignore` file to basically be a superset of our `.gitignore` file to ensure that our production environment is as pristine as our git repository, and I added a test to verify this, so it basically means we can't add to `.gitignore` without also adding to `.dockerignore`.